index.js

```diff
@@ -2,6 +2,18 @@ const humberg = document.querySelector('.navigation');
 const navList = document.querySelector('.nav-list');
 const times = document.querySelector('.times');
 const navLink = document.querySelectorAll('.nav-link');
+const modal = document.querySelector('.modal');
+const btn = document.querySelectorAll('.btn-project');
+
+const modalData = {
+  title: 'Keep track of hundreds of components',
+  description:
+    ' Lorem ipsum dolor sit amet consectetur, adipisicing elit In architecto aliquam accusantium magnam, amet accusamus maxime sit voluptatum laboriosam, nulla optio, repellendus nobis a esse sunt. Commodi amet in corporis!',
+  image: 'images/project-image.png',
+  techinologies: ['Ruby on Rails', 'Css', 'Javascript'],
+  liveVersion: 'https://github.com/ishimwezachee/Portfolio',
+  sourceUrl: 'https://github.com/ishimwezachee',
+};
```

Comment:
- [ ] You have 6 projects on the view but your modals seem to be getting their attributes from only one object. I suggest you have **an array of 6 objects** and then map/loop through the array to display modal details of each object. This will make it easier to accurately display the content of each project modal. It will also make it seamless to add or remove projects in the future.